benches/vector3.rs

```diff
@@ -65,6 +73,10 @@ fn bench_vector3_normalize(c: &mut Criterion) {
         use nalgebra::Vector3;
         bench_unop!(b, op => normalize, ty => Vector3<f32>)
     });
+    // bench_static_math!(group, |b| {
+    //     use static_math::vector3::V3;
+    //     bench_unop!(b, op => normalize, ty => V3<f32>)
+    // });
```

Comment:
What's going on here? Should this be uncommented or removed?

Reply:
yeah my `normalize()` method return a `Result` and in the moment of testing  it gave me an error. You could removed

Reply:
Ah right, I guess it would be good to handle that long term.